packages/@aws-cdk/aws-iot-alpha/lib/logging.ts

```diff
@@ -0,0 +1,130 @@
+import { Resource, Stack, IResource } from 'aws-cdk-lib/core';
+import { Construct } from 'constructs';
+import * as iot from 'aws-cdk-lib/aws-iot';
+import * as iam from 'aws-cdk-lib/aws-iam';
+
+/**
+ * Represents an AWS IoT Logging
+ */
+export interface ILogging extends IResource {
+  /**
+   * The log ID
+   * @attribute
+   */
+  readonly logId: string;
+}
+
+/**
+ * The log level for the AWS IoT Logging
+ */
+export enum LogLevel {
+  /**
+   * ERROR level logging
```

Comment:
In the comments here, you should include the verbiage found [here](https://docs.aws.amazon.com/iot/latest/developerguide/configure-logging.html#log-level) rather than repeating the enum.